### ISSUE TITLE
DIscord crashing when removing friends

### ISSUE THREAD

**Issue by ALazyMeme** [Metalloriff/BetterDiscordPlugins#231]
```
Prettified stacktraces, stack:
 TypeError: Cannot read property 'application_id' of undefined
    at Set.forEach (<anonymous>)
    at GuildAndFriendRemovalAlerts.openModal (betterdiscord://plugins/GuildAndFriendRemovalAlerts.plugin.js:766:143)
    at betterdiscord://plugins/GuildAndFriendRemovalAlerts.plugin.js:672:50
    at Set.forEach (<anonymous>)
    at callOriginalMethod (betterdiscord://plugins/0BDFDB.plugin.js:2319:72)
    at betterdiscord://plugins/0BDFDB.plugin.js:230:15
    at e.module.<computed> [as dispatch] (betterdiscord://plugins/0BDFDB.plugin.js:2334:176) 
Component stack:
     at C (https://discord.com/assets/afa13df16b5ad8339e65.js:1333:78)
    at div
    at T (https://discord.com/assets/71e87549ca9fa8b56538.js:3473:556)
    at AppLayer
    at t (https://discord.com/assets/71e87549ca9fa8b56538.js:3502:176)
    at t (https://discord.com/assets/71e87549ca9fa8b56538.js:4219:854)
    at A (https://discord.com/assets/71e87549ca9fa8b56538.js:3503:560)
    at O (https://discord.com/assets/71e87549ca9fa8b56538.js:3503:397)
    at t (https://discord.com/assets/71e87549ca9fa8b56538.js:3472:761)
    at VerificationLayerProvider
    at t (https://discord.com/assets/71e87549ca9fa8b56538.js:3472:761)
    at AppLayerProvider
    at p (https://discord.com/assets/71e87549ca9fa8b56538.js:2083:545)
    at t (https://discord.com/assets/71e87549ca9fa8b56538.js:718:554)
    at m (https://discord.com/assets/71e87549ca9fa8b56538.js:720:975)
    at t (https://discord.com/assets/71e87549ca9fa8b56538.js:721:436)
    at u (https://discord.com/assets/71e87549ca9fa8b56538.js:3988:170)
    at h (https://discord.com/assets/71e87549ca9fa8b56538.js:699:564)
    at te (https://discord.com/assets/71e87549ca9fa8b56538.js:702:731)
    at t (https://discord.com/assets/71e87549ca9fa8b56538.js:812:541)
    at u (https://discord.com/assets/71e87549ca9fa8b56538.js:3987:99)
    at https://discord.com/assets/71e87549ca9fa8b56538.js:4374:30
    at F (https://discord.com/assets/71e87549ca9fa8b56538.js:5512:700)
    at t (https://discord.com/assets/71e87549ca9fa8b56538.js:5702:962)
    at t (https://discord.com/assets/71e87549ca9fa8b56538.js:5651:817)
    at f (https://discord.com/assets/71e87549ca9fa8b56538.js:2316:813)
```

**Comment by policetonyR** [comment]
I am having the same issue.

**Comment by Metalloriff** [comment]
Fixed in 3.2.0 👍
